eng/scripts/SdkVersion-Common.ps1

```diff
@@ -0,0 +1,15 @@
+# NOTE: Update-PkgVersion and Get-PkgVersion relies on these variables and
+# functions
+$RepoRoot = "${PSScriptRoot}/../.."
+$VersionRegex = '(#define AZURE_\w+_VERSION_MAJOR )(?<major>[0-9]+)(\s+#define AZURE_\w+_VERSION_MINOR )(?<minor>[0-9]+)(\s+#define AZURE_\w+_VERSION_PATCH )(?<patch>[0-9]+)(\s+#define AZURE_\w+_VERSION_PRERELEASE )"(?<prerelease>[a-zA-Z0-9.]*)"';
```

Comment:
We should consider using the regex defined in the SemVer.ps1 script. We can add the extra parts around it if needed.

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/609